Susi/Controllers/AboutViewController/AboutViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  AboutViewController.swift
+//  Susi
+//
+//  Created by Syed on 24/12/18.
+//  Copyright © 2018 FOSSAsia. All rights reserved.
+//
+
+import UIKit
+
+
+class AboutViewController: UIViewController, UITextViewDelegate {
+    
+     let susiUrl = "https://chat.susi.ai"
```

Comment:
Do not define these constants here. Add them in `ControllerConstant`. Also name constant like `susiURL` not `susiUrl`